src/ostree.jsx

```diff
@@ -521,21 +522,17 @@ const DeploymentDetails = (akey, info, packages, doRollback, doUpgrade, doRebase
 };
 
 const DeploymentActions = ({ deploymentIndex, deploymentIsPinned, isCurrent, isStaged }) => {
-    const [isKebabOpen, setKebabOpen] = useState(false);
-
     const togglePin = () => {
         const pinFlags = [];
         if (deploymentIsPinned) {
             pinFlags.push("--unpin");
         }
 
-        cockpit.spawn(["ostree", "admin", "pin", ...pinFlags, deploymentIndex], { superuser: "try" })
-                .then(() => setKebabOpen(false));
+        cockpit.spawn(["ostree", "admin", "pin", ...pinFlags, deploymentIndex], { superuser: "try" });
     };
 
     const deleteDeployment = () => {
-        cockpit.spawn(["ostree", "admin", "undeploy", deploymentIndex], { superuser: "try" })
-                .then(() => setKebabOpen(false));
+        cockpit.spawn(["ostree", "admin", "undeploy", deploymentIndex], { superuser: "try" });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-539-20240305-192913-3e3f669c-fedora-coreos-devel//Coverage/lcov/github-pr.diff.gcov.html)